internal/transport/keepalive_test.go

```diff
@@ -451,13 +399,13 @@ func (s) TestKeepaliveServerEnforcementWithAbusiveClientNoRPC(t *testing.T) {
 
 	serverConfig := &ServerConfig{
 		KeepalivePolicy: keepalive.EnforcementPolicy{
-			MinTime: 2 * time.Second,
+			MinTime: 50 * time.Millisecond,
```

Comment:
This could stay at 2s, right?  Or even 60s..it shouldn't matter as long as it's higher than the client's rate.